src/operations/explore_operation.cpp

```diff
@@ -9,12 +9,20 @@
 #include "util.h"
 
```

Comment:
Could be nice to have a comment at the beginning of the file explaining what is the explore operation goal and how it is working. There you could notably mention how you integrated a point of interest

Reply:
There is a brief introduction to the operations in the `.h` files as well as on Doxygen.